README.md

```diff
@@ -242,3 +306,29 @@ class UserResource extends at.Resource {
 
 ***
 
+### Decorator 
+
+Angular provide a decorator feature to extends a service/factory. Use @decorator to use them and adding some others methods :
+
+```typescript
+@decorator('ngModuleName', 'SomeService')
+class SomeServiceDecorator {
+
+    constructor(
+        @at.inject('SomeService') private parent:SomeService;
+    ) {
+        // do stuff
+    }
+    
+    public someMethod2(anArg: number): boolean {
+        // do some stuff
+        
+       console.log(this.parent.someMethod(anArg));
```

Comment:
Look line 318 @at.inject('SomeService') private parent:SomeService;

"private" keyword on constructor create a private attribute on your instance. Here, parent is the private attribute create on construtor and it values by SomeService.